Client/src/Components/Charts/LineGraph/index.jsx

```diff
@@ -0,0 +1,132 @@
+import {
+	LineChart,
+	Line,
+	XAxis,
+	YAxis,
+	CartesianGrid,
+	Tooltip,
+	Legend,
+	ResponsiveContainer,
+} from "recharts";
+
+import PropTypes from "prop-types";
+import { useTheme } from "@emotion/react";
+import { Stack } from "@mui/material";
+import { useMemo } from "react";
+
+const LineGraph = ({
+	chartData = [],
+	chartConfig = {},
+	styleConfig = {},
+	onClickHandler,
+	onMouseEnterHandler,
+	onMouseMoveHandler,
+	onMouseLeaveHandler,
+}) => {
+	const theme = useTheme();
+
+	const memoizedChartData = useMemo(() => {
+		return chartData.length ? chartData : [];
+	}, [chartData]);
```

Comment:
Here, I believe we don't need useMemo, nor the ternary, since we already have a fallback value for chartData. We can pass chartData straight, without the variable memoizedChartData